### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by ishoborabyose**
What does this PR do?
This is for a page created using BEM

Description of Task to be completed?
files that hold or contain HTML, CSS, and images. 

How should this be manually tested?
After forking this repository, clone it  cd into it, and run the live server

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Medical core</title>
+</head>
+<body>
+    <div class="medical-header__container">
```
Comment (IbrahimBagalwa) [inline index.html]: ## bem naming wrong
this is not bem `medical-header__container`. here you are using element without its block. follow this link [here](https://getbem.com/naming/) for more detail

File: index.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            <ul class="medical-header__list">
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Home</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Product</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Pricing</a></li>
+               <li class="medical-header__item"><a class="medical-header__anchor" href="">Contact</a></li>
+            </ul>
+    
+            <ul class="medical-header__list">
+                <li class="medical-header__item--login"><a class="medical-header__anchor--login" href="">Login</a></li>
+                <li class="medical-header__item--signup"><a class="medical-header__anchor--signup" href="">Join Us</a></li>
```
Comment (IbrahimBagalwa) [inline index.html]: ## bem naming wrong
`medical-header__item--login` , `medical-header__item--signup`, `medical-header__anchor--login` and `medical-header__anchor--signup` this is not bem you are using modifier without its element or block. follow the naming bem link [here](https://getbem.com/naming/).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            </ul>
+        </header>
+
+
+        <div class="medical-banner">
+            <h4 class="medical-banner__header">Welcome</h4>
+            <p class="medical-banner__paragraph--1">A Great Place to Receive Care </p>
+            <p class="medical-banner__paragraph--2">Get your best looking smile now!</p>
```
Comment (IbrahimBagalwa) [inline index.html]: `medical-banner__paragraph--1` `medical-banner__paragraph--2` find a explicite name for your modifier and you are using the modifier without its block or element. this is not bem

File: index.html [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    
+            <ul class="medical-banner__list">
+                <li class="medical-banner__list__item medical-banner__list__item--join"> Join Us</li>
+                <li class="medical-banner__list__item medical-banner__list__item--learn">Learn More</li>
```
Comment (IbrahimBagalwa) [inline index.html]: `medical-banner__list__item` `medical-banner__list__item--join` `medical-banner__list__item--join` `medical-banner__list__item--learn` this is not bem. it was a old way of naming nested element. follow this link [here](https://scalablecss.com/bem-nesting-grandchild-elements/)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+          </p>
+          <a href="#" class="medical-expert__button"
+            >Learn More </a>
+        </div>
+    </div>
+
+    <div class="team-container">
+        <h5 class="team-title">Team</h5>
+        <p class="team-subtitle">Our Team</p>
+        <p class="team-paragraph">Problems trying to resolve the conflict between
```
Comment (IbrahimBagalwa) [inline index.html]: here you are making `team-title` `team-subtitle` `team-paragraph` you put them as a block but I don't see where you are using them

File: index.html [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+                   
+                        <p class="team-card__title">Julian Jameson</p>
+                        <p class="team-card__subtitle"> proffession </p>
+                  
+
+                    <div class="social">
+                        <img class="" src="./assets/facebook (1).svg" alt="">
+                        <img class="" src="./assets/instagram (1).svg" alt="">
+                        <img class="" src="./assets/twitter-1.svg" alt="">
+                    </div>
```
Comment (IbrahimBagalwa) [inline index.html]: tags without class name is not bem. try to find the classname for your `img`

File: style.css [hunk 7]
```diff
@@ -0,0 +1,436 @@
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
```
Comment (IbrahimBagalwa) [inline style.css]: find a class name for this `body`

File: style.css [hunk 8]
```diff
@@ -0,0 +1,436 @@
...
+    padding: 20px 0;
+    font-size: 13px;
+    width: 70%;
+    max-width: 1100px;
+    margin: auto;
+    
+  
+}
+
+.medical-header__logo .medical-header__anchor{
```
Comment (IbrahimBagalwa) [inline style.css]: this is not bem you try to repete your self here `.medical-header__logo .medical-header__anchor`.

File: style.css [hunk 9]
```diff
@@ -0,0 +1,436 @@
...
+
+.medical-header__anchor{
+    text-decoration: none;
+}
+.medical-header__list {
+    display: flex;
+    padding-right: 80px;
+}
+
+.medical-header__list .medical-header__item{
```
Comment (IbrahimBagalwa) [inline style.css]: here as well

File: style.css [hunk 10]
```diff
@@ -0,0 +1,436 @@
...
+    padding: 0 10px;
+    list-style: none;
+    
+}
+
+.medical-header__list .medical-header__item .medical-header__anchor{
```
Comment (IbrahimBagalwa) [inline style.css]: even here `.medical-header__list .medical-header__item .medical-header__anchor`

File: style.css [hunk 11]
```diff
@@ -0,0 +1,436 @@
...
+
+color: #737373;
+
+}
+
+.medical-header__anchor--login{
+    text-decoration: none;
+}
+
+.medical-header__list .medical-header__item--login{
```
Comment (IbrahimBagalwa) [inline style.css]: here as well

File: style.css [hunk 12]
```diff
@@ -0,0 +1,436 @@
...
+    list-style: none;
+}
+.medical-header__list .medical-header__item--login  .medical-header__anchor--login {
```
Comment (IbrahimBagalwa) [inline style.css]: even here

File: style.css [hunk 13]
```diff
@@ -0,0 +1,436 @@
...
+        line-height: 32px;
+        letter-spacing: 0.1px;
+    } 
+
+    .medical-header__anchor--signup{
+        text-decoration: none;
+    }
+    
+
+    .medical-header__list .medical-header__item--signup{
```
Comment (IbrahimBagalwa) [inline style.css]: even here

File: style.css [hunk 14]
```diff
@@ -0,0 +1,436 @@
...
+        list-style: none;
+        width: 80px;
+        margin-left: 20px;
+        background-color: #1BA0AB;
+        text-align: center;
+    }
+
+    .medical-header__list .medical-header__item--signup::after{
```
Comment (IbrahimBagalwa) [inline style.css]: here as well

File: style.css [hunk 15]
```diff
@@ -0,0 +1,436 @@
...
+        content: ' → ';
+        color: white;
+    }
+    .medical-header__list  .medical-header__anchor--signup {
```
Comment (IbrahimBagalwa) [inline style.css]: even here

File: style.css [hunk 16]
```diff
@@ -0,0 +1,436 @@
...
+letter-spacing: 0.2px;
+color: #737373;
+}
+
+.medical-banner__list{
+    display: flex;
+    gap: 10px;
+}
+
+.medical-banner__list .medical-banner__list__item{
```
Comment (IbrahimBagalwa) [inline style.css]: chnge here as well

File: style.css [hunk 17]
```diff
@@ -0,0 +1,436 @@
...
+    display: flex;
+    padding-top: 80px;
+    padding-bottom: 80px;
+    gap: 144px;
+
+    width: 90%;
+    max-width: 1100px;
+    margin: auto;
+}
+.medical-expert__img-wrapper, .medical-expert__body {
```
Comment (IbrahimBagalwa) [inline style.css]: even here

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
## Request Change
keep it up:)
implement the change and re-request the review again. thank you

File: index.html (hunk 1 above)
Comment (mbukeRepo) [reply index.html]: Like Ibrahim said `medical-header__container` should have a parent block called `medical-header`

File: index.html [hunk 18]
```diff
@@ -0,0 +1,144 @@
...
+            <div class="team-card-container">
+                <div class="team-card">
+                    <img class="team-card__img" src="./assets/image-doctor-3.png" alt="">
+                   
+                        <p class="team-card__title">Julian Jameson</p>
+                        <p class="team-card__subtitle"> proffession </p>
+                  
+
+                    <div class="social">
+                        <img class="social-icon" src="./assets/facebook (1).svg" alt="">
```
Comment (mbukeRepo) [inline index.html]: `social-icon` is an element of `social` so use `social__icon`

File: index.html [hunk 19]
```diff
@@ -0,0 +1,144 @@
...
+                        <img class="social-icon" src="./assets/instagram (1).svg" alt="">
```
Comment (mbukeRepo) [inline index.html]: same here

File: index.html (hunk 1 above)
Comment (mbukeRepo) [inline index.html]: `medical-header__container` is an element without a parent block `medical-header`

Comment (mbukeRepo) [reply index.html]: a block comes before an element

**Review by mbukeRepo — COMMENTED** [review]
Congs :clap: :
but please implement these changes

**Comment by ishoborabyose** [comment]
thank you for your feedback.

File: index.html [hunk 20]
```diff
@@ -0,0 +1,145 @@
...
+    <div class="team">
+      <h5 class="banner__header">Team</h5>
+      <p class="team__moto">Our Team</p>
+      <p class="team__description">
+        Problems trying to resolve the conflict between the two major realms of
+        Classical physics: Newtonian mechanics
+      </p>
+
+      <div class="team-card-container">
+        <div class="team-card">
```
Comment (mbukeRepo) [inline index.html]: because this card is located in team section use `team__card` instead of `team-card`

File: index.html [hunk 21]
```diff
@@ -0,0 +1,145 @@
...
+        <p class="expert__description">
+          Problems trying to resolve the conflict between the two major realms
+          of Classical physics: Newtonian mechanics
+        </p>
+        <p class="expert__link">Learn More</p>
+      </div>
+    </div>
+
+    <div class="team">
+      <h5 class="banner__header">Team</h5>
```
Comment (mbukeRepo) [inline index.html]: `banner__header` is an element of `banner` not `team`. use `team__header` in `team` section

File: index.html [hunk 22]
```diff
@@ -0,0 +1,145 @@
...
+      <p class="team__moto">Our Team</p>
+      <p class="team__description">
+        Problems trying to resolve the conflict between the two major realms of
+        Classical physics: Newtonian mechanics
+      </p>
+
+      <div class="team-card-container">
```
Comment (mbukeRepo) [inline index.html]: `team-card-container` is not meaning full may be use `team__cards`

**Review by mbukeRepo — CHANGES_REQUESTED** [review]
kindly make these simple changes

**Comment by ishoborabyose** [comment]
kindly review my code again, thanks.

File: index.html [hunk 23]
```diff
@@ -0,0 +1,145 @@
...
+      <p class="team__moto">Our Team</p>
+      <p class="team__description">
+        Problems trying to resolve the conflict between the two major realms of
+        Classical physics: Newtonian mechanics
+      </p>
+
+      <div class="team__cards">
+        <div class="team__card">
+          <img
+            class="team-card__img"
```
Comment (mbukeRepo) [inline index.html]: no parent block called  `team-card` so `team-card__img` is meaningless. Use `team__img` or `team__card__img`

**Review by mbukeRepo — APPROVED** [review]
LGTM

File: index.html [hunk 24]
```diff
@@ -0,0 +1,129 @@
...
+      </header>
+
+      <div class="banner">
+        <h4 class="banner__header">Welcome</h4>
+        <p class="banner__description">A Great Place to Receive Care</p>
+        <p class="banner__moto">Get your best looking smile now!</p>
+
+        <ul class="banner__btn">
+          <li class="btnn btn--activee">Join Us</li>
+          <li class="btnn btn--disactive">Learn More</li>
```
Comment (IbrahimBagalwa) [inline index.html]: you are using modifier `btn--activee` without its block. or change `btnn` to `btn`

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
fix this then re-request review

**Comment by ishoborabyose** [comment]
Updated.

**Review by IbrahimBagalwa — APPROVED** [review]
## Great Job @ishoborabyose 👍 

LGTM 👍  keep it up :)

**Review by Morrism1 — APPROVED** [review]
Good job! 🚀

**Pull request merged**